portfolio/src/main/webapp/comments.html

```diff
@@ -11,14 +11,17 @@
 <script src="https://kit.fontawesome.com/2e7fd39b04.js" crossorigin="anonymous"></script>
 <script src="https://code.iconify.design/1/1.0.6/iconify.min.js"></script>
 
+<!--Map-->
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAR88Giah8cCEAvT_zDSIREWvgIIAeS8yY"></script>
+
 <!-- Bootstrap CSS only -->
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
 
 <!-- Custom styles -->
 <link rel="stylesheet" href="style.css">
 <link rel="stylesheet" href="comments.css">
 
-<body onload="setUpCommentsPage()">
+<body onload="setUpCommentsPage(); createMap();">
```

Comment:
Might be worth creating a separate load function that calls both `setUpCommentsPage()` and `createMap()` to keep things clean and [separate concerns](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns) as much as possible

Reply:
They're in different files so I assumed it would be okay. I could also just call create map from map.js. what do you think would make the most sense?

Reply:
As long as the functions are defined in global scope they should be accessible between different files. What I do isn't the ground truth or necessarily correct, but I thought it would be cleanest and easiest to maintain if I defined an onLoad() function somewhere and called it from here.

Reply:
I see what you're saying though. I'll move them to the JS files to reduce the calls.

Reply:
Sounds good. Just make sure you're not changing the behavior of the functions by calling them from the files and not from the onload event. Sometimes that can break things if you're relying on DOM elements which might not be loaded when the code is called. A nice middleground is defining a single function that is called from onload that calls all required functions.

Reply:
how do I get that to call map functions and comments functions?

Reply:
actually I guess I could use ordering as a way to basically import functions

Reply:
Yeah, actually best practice I believe is not to even have JS function calls in HTML. Does this work? https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload